### PR TITLE
Fixing Notaro sim out

### PR DESCRIPTION
**Pull request by lawinslow**

### CODE REVIEW

File: R/summarize_var_notaro.R [hunk 1]
```diff
@@ -47,7 +47,7 @@ summarize_var_notaro <- function(nc.file, var.name){
       mutate(doy=as.numeric(DateTime-base.date)/86400+1) %>% 
       select(-DateTime, -tz, -base.date) %>% select(doy, everything()) %>% group_by(doy) %>%  
       summarize_each(c('mean','sd')) %>% 
-      setNames(c('doy',rename_depths(names(.)[-1L]))) %>% gather(key = doy) %>% 
+      setNames(c('doy',rename_depths(names(.)[-1L]))) %>% ##gather(key = doy) %>%
```
Comment (lawinslow) [inline R/summarize_var_notaro.R]: @jread-usgs is `gather` required here? Trying to deal with a weird issue on linux where it consistently fails here.

Comment (jread-usgs) [inline R/summarize_var_notaro.R]: I thought it was, yes. But were you able to get the same results w/o it?

Comment (lawinslow) [inline R/summarize_var_notaro.R]: Yeah, fixed with a later commit.

**Comment by lawinslow** [comment]
NECSC dataset release version.

**Pull request merged**